config/locales/en.yml

```
@@ -12,48 +12,57 @@ en:
       extra_user_fields:
         menu:
           title: Manage extra user fields
+    admin_log:
+      organization:
+        export_users: "%{user_name} has exported the users"
```

```suggestion
        export_users: "%{user_name} has exported the users from extra user fields export feature"
```

I think we should be more precise on the feature which allows to export because it is an optional module